src/reanimated2/Hooks.js

```diff
@@ -343,6 +353,16 @@ export function useAnimatedStyle(updater, dependencies) {
 // when you need styles to animated you should always use useAS
 export const useAnimatedProps = useAnimatedStyle;
 
+export function useAnimatedPropAdapter(adapter, nativeProps) {
```

Comment:
If it's not a hoot why it is in the file with hooks?

Reply:
fixed f5a7d01